pkg/storage/storage.go

```diff
@@ -0,0 +1,73 @@
+package storage
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type Profile struct {
+	Name string `json:"name"`
+	Key  string `json:"key"`
+}
+
+type Storage struct {
+	Default     string     `json:"default"`
```

Comment:
It's gonna be Default profile name, right? So let's call it such way: DefaultProfileName or DefaultProfile.
Otherwise it looks like it indicates where something is default or not so from the first look it asks to be named like IsDefault which wasn't your case.